bokehjs/src/coffee/models/annotations/label.coffee

```diff
@@ -143,6 +144,7 @@ class LabelView extends Renderer.View
         'font-size': "#{@_text_font_size[i]}"
         'font-family': "#{@mget('text_font')}"
         'background-color': "#{@visuals.background_fill.color_value()}"
+        'transform': "rotate(#{rad_to_deg(@_angle[i])}deg)"
```

Comment:
I'm not sure why convert to degrees here. `rotate()` transform supports radians and other (see https://developer.mozilla.org/en-US/docs/Web/CSS/angle).